tests/lib/rules/eol-last.js

```diff
@@ -26,10 +27,22 @@ ruleTester.run("eol-last", rule, {
         "var a = 123;\n\n",
         "var a = 123;\n   \n",
 
-        "\r\n",
```

Comment:
I think we should leave these tests - even though we we're not enforcing a Windows linebreak style any longer, we want to make sure that this rule is still compatible with these cases